### PR TITLE
fix minimum required space for data directory

### PR DESCRIPTION
**Pull request by slightlyskepticalpotat**
The minimum required space for the data directory (shown when first launching the qt wallet) was originally unchanged from Bitcoin. This reduces it to just above the current size of the Dogecoin blockchain.

### CODE REVIEW

**Comment by DemonRx** [comment]
Is this intro statement even necessary? Being that the standard storage medium in the average home PC is 500gb+. It'll be probably more than a decade before that might be even reached.

Or could this be made dynamic so it updates the statement to the user automatically?  As it seems redundant having to update this value constantly.

**Comment by DemonRx** [comment]
@slightlyskepticalpotat the commit name says 'maximum' vs your PR says 'minimum' probably want to fix that typo.

**Comment by slightlyskepticalpotat** [comment]
Thanks for catching that, I've fixed it. I think the intro statement is still needed now, because having a core wallet taking up 1/10 of your hard drive is still a lot, especially if you could use a lighter wallet. Additionally, downloading 50 GB of data still takes a while.

**Comment by rnicoll** [comment]
Verified disk usage on one of my nodes is ~46GB so this gives a working overhead for now.

**Review by rnicoll — APPROVED** [review]

**Comment by shibe2** [comment]
With txindex it occupies about 52 GB on my disk. Note that decimal prefix is used here.

**Comment by rnicoll** [comment]
> Or could this be made dynamic so it updates the statement to the user automatically? As it seems redundant having to update this value constantly.

In terms of updates, we have to review patches as we update each release anyway, so we'll review then. Although I'm _hoping_ a later Bitcoin Core makes this depend on block max size & checkpoints or something.

File: src/qt/intro.cpp [hunk 1]
```diff
@@ -23,7 +23,7 @@
 
 static const uint64_t GB_BYTES = 1000000000LL;
 /* Minimum free space (in GB) needed for data directory */
-static const uint64_t BLOCK_CHAIN_SIZE = 120;
+static const uint64_t BLOCK_CHAIN_SIZE = 50;
```
Comment (rnicoll) [inline src/qt/intro.cpp:26]:
```suggestion
static const uint64_t BLOCK_CHAIN_SIZE = 55;
```
Lets have a bit more space just to be sure

Comment (slightlyskepticalpotat) [reply src/qt/intro.cpp:26]: It uses 47 GB on my node, but I don't have txindex.

**Review by patricklodder — APPROVED** [review]
I think this is good for this release.

**Pull request merged**